tests/processors/pfizer/test_extractors.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import datetime
+import pytest
+import processors.pfizer.extractors as extractors
+
+
+class TestPfizerExtractors(object):
+    def test_stub_record_is_valid(self, stub_record):
+        extractors.extract_trial(stub_record)
+
+    @pytest.mark.parametrize('age,expected_min_age,expected_max_age', [
+        ('12 Years and older', '12 Years', None),
+        ('1 Year - 20 Years', '1 Year', '20 Years'),
+        ('1 Month - 3 Months', '1 Month', '3 Months'),
+        ('1 Week - 12 Weeks', '1 Week', '12 Weeks'),
+        ('1 Day - 12 Days', '1 Day', '12 Days'),
+        ('1 Hour - 72 Hours', '1 Hour', '72 Hours'),
+        ('1 Minute - 60 Minutes', '1 Minute', '60 Minutes'),
+        ('0 Months - 60 Years', '0 Months', '60 Years'),
+        ('N/A and older', None, None),
+        ('N/A - N/A', None, None),
+        ('N/A - 5 Years', None, '5 Years'),
+        ('1 Year - N/A', '1 Year', None),
+        ('', 'Unknown', 'Unknown'),
+        (None, 'Unknown', 'Unknown'),
+    ])
+    def test_extracts_minimum_and_maximum_ages(self, stub_record, age, expected_min_age, expected_max_age):
+        stub_record.update({
+            'age_range': age,
+        })
+
+        trial = extractors.extract_trial(stub_record)
+
+        assert trial['age_range'] == {
+            'min_age': expected_min_age,
+            'max_age': expected_max_age,
+        }
+
+    def test_minimum_and_maximum_age_are_unknown_by_default(self, stub_record):
```

Comment:
Isn't this already covered [in the test above](https://github.com/opentrials/processors/blob/52ac6749d92c7dfe57ca6ded32c7af05b3c4d238/tests/processors/pfizer/test_extractors.py#L30)?